routes/web.php

```diff
@@ -34,7 +34,9 @@
 
 
 
-
+ 
+Route::get('/photovoltaic-2','App\Http\Controllers\experimentController@vicelabexp2')->name('vicelab-exp2');
```

Comment:
Also check the part URL

Reply:
Like we discussed, @saidabdul80, you can use the new way of routing 
Like:
Route::post('/photovoltaic', [experimentController::class, 'vicelabexp2']);